core/domain/user_jobs_one_off.py

```diff
@@ -1073,3 +1073,52 @@ def reduce(key, values):
 
         if len(values) != 1:
             yield ('FAILURE', values)
+
+
+class OldDraftDeletionJob(jobs.BaseMapReduceOneOffJobManager):
```

Comment:
The deletion keyword in this job sounds like we are deleting any model, should we use the "discard" word instead?